source/_docs/blueprint/selectors.markdown

```diff
@@ -267,19 +267,24 @@ color_temp:
 ```
 
 {% configuration color_temp %}
-min_mireds:
-  description: The minimum color temperature in mireds.
+unit:
+  description: Unit to use for the selector. Either "Kelvin" or "Mired".
+  type: string
+  defaulr: "Kelvin"
+  required: true
+min:
+  description: The minimum color temperature.
   type: integer
-  default: 153
+  default: 2700 for Kelvin, 153 for Mired
   required: false
-max_mireds:
-  description: The maximum color temperature in mireds.
+max:
```

Comment:
Left a comment on the parent PR as well, this is breaking existing blueprints (or scripts) using this selector. We cannot do that.

../Frenck